include/tdi/common/tdi_table.hpp

```diff
@@ -29,7 +29,7 @@
 #include <unordered_map>
 
 #include <tdi/common/tdi_defs.h>
-#include <tdi/common/tdi_table_info.hpp>
+//#include <tdi/common/tdi_table_info.hpp>
```

Comment:
Why not remove this line instead of commenting out, the include file is removed in this pull req....

Reply:
More changes regarding this will be coming in soon :)